# try-catch-core
My use-case's async tryCatch. The callback is securely wrapped with a `dezalgo` and `once`.
